### PR TITLE
chore: Adds CodeTour to the RAG JS and TS templates

### PR DESCRIPTION
**Pull request by waldekmastykarz**

### CODE REVIEW

**Review by MuyangAmigo — APPROVED** [review]

File: templates/js/custom-copilot-rag-microsoft365/.tours/load-data-from-graph-connector.tour [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+      "description": "### Update Entra app manifest\n\nChange the permission to be able to read external items. This is necessary to grant your app access to external items imported to Microsoft 365 by the Microsoft Graph connector. Change the permission to:\n\n```text\nExternalItem.Read.All\n```",
+      "line": 24,
+      "selection": {
+        "start": {
+          "line": 24,
+          "character": 28
+        },
+        "end": {
+          "line": 24,
+          "character": 42
```
Comment (MuyangAmigo) [inline templates/js/custom-copilot-rag-microsoft365/.tours/load-data-from-graph-connector.tour:16]: Character selection seems incorrect?

image

File: templates/js/custom-copilot-rag-microsoft365/.tours/load-data-from-graph-connector.tour [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+        }
+      },
+      "title": "Update Entra app manifest"
+    },
+    {
+      "file": "src/app/app.js",
+      "description": "### Update default scopes\n\nUpdate the scopes that the Microsoft Graph client should request when connecting to Microsoft Graph. This is necessary for your app to be able to read external items imported by the Graph connector. Change the permission to:\n\n```text\nExternalItem.Read.All\n```",
```
Comment (MuyangAmigo) [inline templates/js/custom-copilot-rag-microsoft365/.tours/load-data-from-graph-connector.tour:23]: Our templates have a ts version also, can we create one for ts file?

Comment (waldekmastykarz) [reply templates/js/custom-copilot-rag-microsoft365/.tours/load-data-from-graph-connector.tour:23]: Absolutely! I wanted to start with one to check if we agree with the approach. I'll go on and create the same for the TS template

File: templates/js/custom-copilot-rag-microsoft365/.vscode/extensions.json [hunk 3]
```diff
@@ -1,5 +1,6 @@
 {
   "recommendations": [
-    "TeamsDevApp.ms-teams-vscode-extension"
+    "TeamsDevApp.ms-teams-vscode-extension",
+    "vsls-contrib.codetour"
```
[the remark this answers is not in the document]
Comment (waldekmastykarz) [reply templates/js/custom-copilot-rag-microsoft365/.vscode/extensions.json:4]: Good catch. Sorry I missed it

**Pull request merged**